tests/support/server.rs

```diff
@@ -23,6 +23,7 @@ pub struct Txn {
     pub read_timeout: Option<Duration>,
     pub response_timeout: Option<Duration>,
     pub write_timeout: Option<Duration>,
+    pub chunk_size: Option<usize>,
```

Comment:
That's the point. It's blocking the response to force `WouldBlock`. It doesn't necessarily need to be chunked encoding because that isn't what it's testing.